web/src/components/storage/MenuDeviceDescription.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) [2023-2025] SUSE LLC
```

Comment:
NP: should it be [2025] only?

Reply:
No. It's code moved from another file. The original code is not new.